spacepy/toolbox/__init__.py

```diff
@@ -1102,8 +1112,12 @@ def get_url(url, outfile=None, reporthook=None, cached=False,
 
     This is similar to the deprecated ``urlretrieve``.
 
+    .. versionchanged:: 0.2.3
+       In earlier versions of SpacePy invalid combinations of cached and outfile raised RuntimeError,
+       changed to ValueError.
```

Comment:
Line too long, docstrings should wrap at 72 per PEP8.

Reply:
Updated.